lua/usip_player.lua

```diff
@@ -830,7 +830,7 @@ function genMultiUsips(strTmpFolderPath, tUsipConfigDict)
     local tUsipNames
     -- list of all generated usip file paths
     if tFlasherHelper.fStoreTempFiles then
-        tResult, aDataList, tUsipNames = tUsipGen:gen_multi_usip_hboot(tUsipConfigDict, strTmpFolderPath)
+        tResult, tUsipNames, aDataList = tUsipGen:gen_multi_usip_hboot(tUsipConfigDict, strTmpFolderPath)
     else
         aDataList, tUsipNames = tUsipGen:gen_multi_usip(tUsipConfigDict)
         tResult = true
```

Comment:
This could be cleaned up... but not now. Both should return the values in the same order, and tResult is unused.